packages/components/pages/f-checkout/test/component/f-checkout.component.spec.js

```diff
@@ -5,6 +5,8 @@ const Checkout = require('../../test-utils/component-objects/f-checkout.componen
 let checkout;
 let checkoutInfo;
 
+const orderTime = 'Wednesday 01:45';
```

Comment:
Unless I'm wrong, this is only used in one test at the moment, so we can move this `const` to the scope of that test rather than having it as a global variable.